### ISSUE TITLE
create_release error

### ISSUE THREAD

**Issue by ibennetch** [phpmyadmin/phpmyadmin#13613]
When running `./scripts/create_release 4.7.4 QA_4_7 --tag --stable` I get this error:

```
   Dont forget to push tags using: git push --tags
fatal: pathspec 'composer.lock' did not match any files
```

This probably comes from the command `git rm --force composer.lock`

**Comment by nijel** [comment]
Did it work fine for 4.7.3? Did you upgrade git meanwhile?

Looking at the QA_4_7 branch, the file has been added for tag, but the removal later fails, what sounds a bit strange....

**Comment by ibennetch** [comment]
I believe it's been happening since we integrated Composer in to the release process.

**Comment by nijel** [comment]
What git version do you use? Maybe there is some change in handling ignored files...

**Comment by ibennetch** [comment]
git version 2.7.0.GIT

This doesn't make much sense; I've done some tests in a new and empty repository.

```
echo "foo" > .gitignore
touch foo
git add --force foo
git commit "Add"
git rm --force foo
git commit -sm "Remove"
```
works as expected and without error.

I'll have to add some debugging code to see what's going on here.

**Comment by nijel** [comment]
Did the error happen now? Or it worked just fine?

**Comment by ibennetch** [comment]
It happened again.

I'm using a more updated Git now, too (though I don't recall the version off hand).

**Comment by nijel** [comment]
Ah, now I see it - we're operating on wrong branch here...

**Comment by nijel** [comment]
BTW: It seems you forgot to push the RELEASE_4_7_6 tag...

**Comment by ibennetch** [comment]
Thanks for finding and fixing this.

It looks to me like the tag was pushed correctly, including at https://github.com/phpmyadmin/phpmyadmin/tags and with `git ls-remote --tags origin`. Might you see it now?

**Comment by nijel** [comment]
Hmm, apparently something is broken in my working copy - I can indeed see it in `git ls-remote --tags origin`, but the tag is not fetched. Anyway the repository looks okay, it's problem on my side, sorry for noise.

**Comment by nijel** [comment]
Hmm, it looks strange - the tag is not on the QA_4_7 branch, but on seraparate one split from QA_4_7 just before release (last common commit is c6f3c4002c56509c8699ab38e123eff45778552d). 

See the two histories:

* https://github.com/phpmyadmin/phpmyadmin/commits/39a20c08156dd2e3c2704ba75db5bafa85efbaf0
* https://github.com/phpmyadmin/phpmyadmin/commits/RELEASE_4_7_6

Anyway `git fetch --tags` did get me the missing tags and I've merged the split history in 79a95b616aa404990c8cd12a039604477bb6d2b2.
